CONTRIBUTING.md

```diff
@@ -31,6 +31,14 @@ Participation in this project comes under the [Contributor Covenant Code of Cond
 - Once review has occurred, please rebase your PR down to a single commit. This will ensure a nice clean Git history.
 - Finally - *Thanks* for considering submitting code to Agones!
 
+## Formatting
+
+When submitting pull requests, make sure to do the following:
+
+- Format all Go code with [gofmt](https://golang.org/cmd/gofmt/).
```

Comment:
`gofmt` and/or `goimport` ?

Reply:
Hmm. Technically `gofmt`, but you'd be mad to write your own import statements and not use `goimports`.

Reply:
How about another sentence after this one that says, "Many people use `goimports` which adds import statements automatically while also running `gofmt`"?

Reply:
SGTM.